### DISCUSSION TITLE
Music player

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#28774] [Ideas] [answered]
I have noticed that some players choose to listen to music instead of playing when relaxing, and there are some small details that can greatly help these players :)               

---These are useful for everyone
- Add a "thumbnail" mode to display optional songs in full screen, which can display more songs on one screen. And this is also beneficial for ordinary players, not just the less than ten on the list on the right.
- Collections can be deleted, transferred, and added in batches, which can be quickly operated in conjunction with the "Thumbnail" mode above.
- In "Now playing", add a thumbnail to each song in the song list for easy recognition. Because I noticed that there are too many blank areas in this list.
- Special searches should be specifically labeled, such as "CS<5", using other colors.
- Hmm... If its RAM usage is even lower, it can defeat cloudmusic music software in all aspects.
- Allow players to modify the appearance of Beatmap's cover, MV, title, and other visual elements (isolated from the original for gameplay). This is also beneficial for searching (new title).

---Here are other possible additions that are entirely for listening to music
- Do not press F6 again on this big pink circle screen to switch music. When the big pink circle is exclusively on the screen, audio controls will be displayed below, including the music progress bar, "manage collections" button, and "thumbnail" mode button. However, don't worry, the purpose of F6 is to control audio on other interfaces (such as in multiplayer mode).
- Perhaps a "< Back" button can be added to the main menu to go back to the big pink circle screen.
- MV can be played on the big pink circle screen.
- A relatively simple way to adjust sound is to make the mods list accessible to players anywhere (shortcut key), and use mods to adjust music. do not automatically deselect mods.

**Comment by bdach** [comment, marked as answer]
osu! is not and will not be a music player.
